README.md

```diff
@@ -113,6 +118,34 @@ To set up Chromium remote debugging, edit project build/launch configurations in
 
 Next, launch the game and then open a separate Chrome/Chromium browser and navigate to `chrome://inspect` where your app should be listed, sometimes intermittently or after a delay, under the `Remote Target #localhost` section.  Click `inspect` to launch the remote debugging session.
 
+## Release Unreal Immutable SDK Plugin
+
+1. Release the plugin using github releases. See https://github.com/immutable/unreal-immutable-sdk/releases
+
+## Release Blui Plugin for UE4
+
+For UE4 we are using Blui as the in built browser does not work.
+
+1. Download this release supporting UE 4.27 and 4.26 https://github.com/getnamo/BLUI-Unreal/releases/tag/4.2.0
+2. Download the CEF 90 Browser from https://github.com/getnamo/BLUI-Unreal/releases/tag/v4.6.0 (`BluBrowserCEF....7z`)
+3. Delete the Thirdparty/cef/Win folder from the BLUI plugin
+4. Place the Win folder from the `BluBrowserCEF...7z` into Thirdparty/cef
+5. Disable cefdebug settings in `Blu.cpp`, following line
+    ```
+    // BluManager::Settings.remote_debugging_port = 7777;
+    ```
+6. Zip the Blui plugin folder and upload to the github release.
+
+## How to Add Game Bridge Asset to Packaged Resources
```

Comment:
Maybe this part should be in the CONTRIBUTING.md file instead. But can be done in another PR.

Reply:
Not sure if `CONTRIBUTING.md` is the right place too? May be a bunch of How to guides and have them linked in readme